client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -129,317 +118,140 @@ class CancelPurchaseForm extends React.Component {
-				placeholder={ translate( 'How can we improve our upgrades?' ) }
-			/>
-		);
-		reasons.onlyNeedFree = (
-			<FormLabel key="onlyNeedFree">
-				<FormRadio
-					name="onlyNeedFree"
-					value="onlyNeedFree"
-					checked={ 'onlyNeedFree' === this.state.questionOneRadio }
-					onChange={ this.onRadioOneChange }
-				/>
-				<span>{ translate( 'The plan was too expensive.' ) }</span>
-				{ 'onlyNeedFree' === this.state.questionOneRadio && onlyNeedFreeInput }
-			</FormLabel>
+		appendRadioOption(
+			'onlyNeedFree',
+			translate( 'The plan was too expensive.' ),
+			translate( 'How can we improve our upgrades?' )
 		);
 
-		const anotherReasonOneInput = (
-			<FormTextInput
-				className="cancel-purchase-form__reason-input"
-				name="anotherReasonOneInput"
-				value={ this.state.questionOneText }
-				onChange={ this.onTextOneChange }
-				id="anotherReasonOneInput"
-			/>
+		appendRadioOption(
+			'couldNotActivate',
+			translate( 'I was unable to activate or use the product.' ),
+			translate( 'Where did you run into problems?' )
 		);
-		reasons.anotherReasonOne = (
-			<FormLabel key="anotherReasonOne">
-				<FormRadio
-					name="anotherReasonOne"
-					value="anotherReasonOne"
-					checked={ 'anotherReasonOne' === this.state.questionOneRadio }
-					onChange={ this.onRadioOneChange }
-				/>
-				<span>{ translate( 'Another reason…' ) }</span>
-				{ 'anotherReasonOne' === this.state.questionOneRadio && anotherReasonOneInput }
-			</FormLabel>
+
+		appendRadioOption(
+			'noLongerWantToTransfer',
+			translate( 'I no longer want to transfer my domain.' )
 		);
 
-		// Survey questions only for Jetpack subscriptions
-		const couldNotActivateInput = (
-			<FormTextInput
-				className="cancel-purchase-form__reason-input"
-				name="couldNotActivateInput"
-				id="couldNotActivateInput"
-				value={ this.state.questionOneText }
-				onChange={ this.onTextOneChange }
-				placeholder={ translate( 'Where did you run into problems?' ) }
-			/>
+		appendRadioOption(
+			'couldNotCompleteTransfer',
+			translate( 'Something went wrong and I could not complete the transfer.' )
 		);
-		reasons.couldNotActivate = (
-			<FormLabel key="couldNotActivate">
-				<FormRadio
-					name="couldNotActivate"
-					value="couldNotActivate"
-					checked={ 'couldNotActivate' === this.state.questionOneRadio }
-					onChange={ this.onRadioOneChange }
-				/>
-				<span>{ translate( 'I was unable to activate or use the product.' ) }</span>
-				{ 'couldNotActivate' === this.state.questionOneRadio && couldNotActivateInput }
-			</FormLabel>
+
+		appendRadioOption(
+			'useDomainWithoutTransferring',
+			translate( 'I’m going to use my domain with WordPress.com without transferring it.' )
 		);
 
-		const { questionOneOrder } = this.state,
-			orderedReasons = questionOneOrder.map( question => reasons[ question ] );
+		appendRadioOption(
+			'anotherReasonOne',
+			translate( 'Another reason…' ),
+			translate( 'How can we improve?' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'How can we improve our upgrades?' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).